### ISSUE TITLE
Improve accessibility of the toolbar

### ISSUE THREAD

**Issue by tim-schilling** [jazzband/django-debug-toolbar#1842]
We should start identifying what UI changes we need to support WCAG 2.2.

A tool that can help identify issues is: https://accessibilityinsights.io/

Thibaud has a good talk on accessibility from a few years ago: https://www.youtube.com/watch?v=PO0w2BJk33s

**Comment by sabderemane** [comment]
Hi there, I'll make an accessibility audit and see with the Django Accessibility Team to break it into tickets and provide help on that :)

**Comment by thibaudcolas** [comment]
FYI @sabderemane @knyghty discussed this – we’re keen to try to do this with a Djangonaut Space participant. @sabderemane will do the audit and let them do the fixes if no one wants to audit. Or they can do it all.

**Comment by tim-schilling** [comment]
Sounds good, I'll bring it up with the group next week and let you know. We'll probably take some of the logistical discussion offline, but put the long term stuff here.

**Comment by sabderemane** [comment]
Thank you @thibaudcolas ❤️

**Comment by VeldaKiara** [comment]
Based on the incoming design, I ran automated accessibility tests with axe-core 4.10.2 (WCAG 2.0/2.1 A + AA rulesets) against the rendered toolbar in both themes:</p>
  | Light | Dark
-- | -- | --
Violations | 0 | 0
Passes | 11 | 11
Incomplete | 0 ✅ | 0 ✅
Inapplicable | 49 | 49

A second scan against populated panels also reports 0 violations in both themes.
